spec/presenters/quote_presenter_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper'
+
+describe Spree::QuotePresenter do
+  include Spree::BaseHelper
+  let(:quote) { create(:quote) }
+  let(:quote1) { create(:quote, author_name: 'abc') }
+
+  describe "#author_name" do
+    context 'when author_name is blank' do
+      it "returns decorated author_name" do
```

Comment:
when author_name is blank then it returns "Anonymous" and when author_name is present then it returns author_name itself